canonical-web-and-design/maas-ui
# refactor(ui): refactor machine overview card to work with controllers

## Done

- Refactored machine overview card to work with controllers

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the React summary tab of a controller
- Check that the overview card looks like [the design](https://docs.google.com/document/d/1H-GWCLx3aSRiJRo9h5Fz-6IFLQ3adsa63pRFGlBFuq0/edit#heading=h.gczn4qs17xmf) (status portion will be built in a follow up)
- Check that the React machine summary tab still works as before

## Fixes

Fixes canonical-web-and-design/app-tribe#1000

## Screenshot

![Screenshot 2022-06-01 at 16-40-27 bolla summary bolla MAAS](https://user-images.githubusercontent.com/25733845/171343316-6626a221-6d28-4288-b4a3-bdfbb33d5400.png)
